Traceback (most recent call last):
  File "fitdadi_infer_DFE.py", line 593, in <module>
    DemographicAndDFEInference().main()
  File "fitdadi_infer_DFE.py", line 311, in main
    os.mkdir(outdir)
OSError: [Errno 2] No such file or directory: '../20201208_eduardo_sfs/AW_n6_norel'
